Docu update on emission driven compset and NorESM cmip6 data with different physics (#355)
NorESMhub/NorESM: doc/configurations/experiments.rst

For an overview of the compsets provided for CESM2, please see: http://www.cesm.ucar.edu/models/cesm2/config/compsets.html.

Emission-driven compsets
''''''''''''''''''''''''
NorESM2 can be run in emission-driven mode for interactive carbon-cycle studies. Currently, this configuration is only supported for the LM-resolution. In order to run NorESM2-LM in emission-driven mode, the biogeochemical physics ``[_BGC]%pys`` is set to  ``%BPRPDMS`` (instead of ``%BDRDDMS``)

There exists some predefined-emission driven compsets which usually ends with **esm** :

* **N1850esm**: Emission driven coupled configuration for NorESM for pre-industrial (1850) conditions

> * **NHISTesm**: Emission driven historical configuration from 1850 up to year 2015 

Author: ```suggestion
* **NHISTesm**: Emission driven historical configuration from 1850 to year 2014 
```

> 

Reviewer: replace 2015 with 2014.
You can also add:
* **NSSP585esm**: Emission driven SSP5-8.5 scenario configuration from 2015 up to year 2100
* **NSSP534esm**: Emission driven SSP5-3.4 scenario configuration from 2015 up to year 2100

Author: Done!

> 

Author: ```suggestion
* **NSSP585esm:** Emission driven SSP5-8.5 scenario configuration from 2015 to year 2100
* **NSSP534esm:** Emission driven SSP5-3.4 scenario configuration from 2015  to year 2100
```

For an overview of the compsets provided for CESM2, please see: http://www.cesm.ucar.edu/models/cesm2/config/compsets.html.

Emission-driven compsets
''''''''''''''''''''''''
NorESM2 can be run in emission-driven mode for interactive carbon-cycle studies. Currently, this configuration is only supported for the LM-resolution. In order to run NorESM2-LM in emission-driven mode, the biogeochemical physics ``[_BGC]%pys`` is set to  ``%BPRPDMS`` (instead of ``%BDRDDMS``)

There exists some predefined-emission driven compsets which usually ends with **esm** :

* **N1850esm**: Emission driven coupled configuration for NorESM for pre-industrial (1850) conditions
* **NHISTesm**: Emission driven historical configuration from 1850 to year 2014 

> * **NSSP585esm:** Emission driven SSP5-8.5 scenario configuration from 2015 to year 2100

Author: ```suggestion
* **NSSP585esm**: Emission driven SSP5-8.5 scenario configuration from 2015 to year 2100
```

> * **NSSP534esm:** Emission driven SSP5-3.4 scenario configuration from 2015  to year 2100

Author: ```suggestion
* **NSSP534esm**: Emission driven SSP5-3.4 scenario configuration from 2015  to year 2100
```

> * **NSSP534esm**: Emission driven SSP5-3.4 scenario configuration from 2015  to year 2100

Author: ```suggestion
* **NSSP534esm**: Emission driven SSP5-3.4 scenario configuration from 2015  to year 2100

```
